### ISSUE TITLE
Not working with static binaries

### ISSUE THREAD

**Issue by Kiblyn11** [arget13/DDexec#11]
I was trying to run CDK tool with DDExec but it fails to get system page size.

```
base64 -w0 /dev/shm/cdk | bash ddexec.sh eva --full
fatal error: failed to get system page size
runtime: panic before malloc heap initialized

runtime stack:
runtime.throw(0xa4fb42, 0x1e)
        /opt/hostedtoolcache/go/1.15.15/x64/src/runtime/panic.go:1116 +0x72 fp=0x7ffda32a1ea0 sp=0x7ffda32a1e70 pc=0x435b72
runtime.mallocinit()
        /opt/hostedtoolcache/go/1.15.15/x64/src/runtime/malloc.go:438 +0x385 fp=0x7ffda32a1ec8 sp=0x7ffda32a1ea0 pc=0x40c6c5
runtime.schedinit()
        /opt/hostedtoolcache/go/1.15.15/x64/src/runtime/proc.go:563 +0x65 fp=0x7ffda32a1f20 sp=0x7ffda32a1ec8 pc=0x4394c5
runtime.rt0_go(0x7ffda32a1f4f, 0x1, 0x7ffda32a1f4f, 0x169622f6f672f3d, 0xf100000000000000, 0x7ffda32a1f, 0x0, 0x300000000000000, 0x4000000000000000, 0x400000000004000, ...)
        /opt/hostedtoolcache/go/1.15.15/x64/src/runtime/asm_amd64.s:214 +0x125 fp=0x7ffda32a1f28 sp=0x7ffda32a1f20 pc=0x4691c5
```

Are golang bin impossible to run with this technique ?

**Comment by arget13** [comment]
Hmm, it should work with go binaries, that bug was explicitly fixed in #5. I'll look into it. Thanks.

**Comment by Kiblyn11** [comment]
Working great thanks !
